libs/designsystem/modal/experimental/src/modal/modal.component.ts

```diff
@@ -3,6 +3,9 @@ import { IonContent, IonModal } from '@ionic/angular';
 import { OverlayEventDetail } from '@ionic/core/components';
 import { KirbyAnimation } from '@kirbydesign/designsystem/helpers';
 
+type Size = 'xxs' | 'xs' | 'sm' | 'md' | 'lg';
+type SizeTemp = 'md';
```

Comment:
We will soon get the correct t-shirt sizes, but for now `md` is the default size.

Reply:
ok